governance/api.go

```diff
@@ -331,7 +331,26 @@ func getParams(governance Engine, num *rpc.BlockNumber) (map[string]interface{},
 	if err != nil {
 		return nil, err
 	}
-	return pset.StrMap(), nil
+	sm := pset.StrMap()
+
+	// Hide some parameters that are deprecated since hardforks.
+	// Reduces the confusion like, e.g., stakingupdateinterval is shown as 86400 but actually irrelevant.
+	rule := governance.BlockChain().Config().Rules(new(big.Int).SetUint64(blockNumber))
+	if rule.IsKore {
+		// Gini option deprecated since Kore, as All committee members have an equal chance
+		// of being elected block proposers.
+		delete(sm, "reward.useginicoeff")
```

Comment:
Rather than a literal string, how do you feel `GovernanceKeyMapReverse[params.UseGiniCoeff]`?

Reply:
Let's do that in the upcoming kaiax/gov module @ian0371. Thanks for the suggestion